### DISCUSSION TITLE
Differences between images

### DISCUSSION THREAD

**Discussion by tino3no** [ThatGuySam/vimeo-thumbnail-server#45] [Q&A]
Curious.  I changed the thumbnail on Vimeo.  On Vumbnail the Responsive image is blank, JPG is blank, JPG Large is the old thumbnail, and JPG Medium and Small are the correct new thumbnail.

**Comment by ThatGuySam** [comment]
It's the caching. 

You should be able to just add a new query string like: ?v2

Just be nice and don't abuse it or I'll have to block your site or turn on query string caching for everyone
